UniversalAlgebra/Varieties/Closure.lagda

```diff
@@ -168,7 +168,7 @@ module _ {α β : Level}{𝒦 : Pred (Algebra α 𝑆)(ov α)} where
  subalgebra→S {𝑩} (𝑨 , ((𝑪 , C≤A) , KA , B≅C)) = ssub sA B≤A
   where
    B≤A : 𝑩 ≤ 𝑨
-   B≤A = ≤-iso 𝑨 B≅C C≤A
+   B≤A = A≅B→B≤C→A≤C {𝑪 = 𝑨} B≅C C≤A
```

Comment:
That middle `→` is not a good name. Yes, that's the type, because of currying. But the semantics here is that of 'and'. So either \x or \and would be better (and what the library uses as convention).

Reply:
Okay, got it.  I'll change the name now.  Thanks!

Reply:
Would you be okay with \And or \and, as in,

```
A≅B⋀B≤C→A≤C 
```
or
```
A≅B∧B≤C→A≤C
```
If so, which do you like better?

Feel free to suggest others.

Personally, I think it's pretty ugly to have the letters A, B, and C in the name of the theorem.  However, I have found that such theorems, and how to apply them, are nearly impossible to remember without such names.

Reply:
Actually, your first suggestion `A≅B×B≤C→A≤C` is starting to grow on me. :)

Reply:
I'm kind of partial to `\x`, this being constructive and all.

I'm ok with either having letters, or going with `≅x≤→≤` (but with the proper x which I can't type right now).  But if the choice is between different ands, I prefer \and.

Reply:
Sounds reasonable.  Thanks!